packages/core/telemetry/admin/index.js

```diff
@@ -0,0 +1,24 @@
+import { ampli } from './src/ampli/index.js'; // eslint-disable-line
+
+// Loads ampli sdk for the correct environment, checks if processing should be disabled or not
+if (!ampli.amplitude) ampli.load({ environment: 'production' });
```

Comment:
- if statements should not be on a single line
- Can you explain that it does the comment isn't really helping at the moment?

Reply:
My mistake here, that comment is not relevant at the moment, I'll remove it and refactor the if statement